### PR TITLE
updates cloud security section of billing page

### LINKED ISSUE #5166: [Enhancement]: Serverless Cloud Protection Billing Update
### Description

As per this EPIC- **https://github.com/elastic/security-team/issues/8970**
we are charging customers only for the Compute, Database and Storage assets. We need to elaborate on this definition of Billable assets so that customers are aware of what exactly we are charging them for, while going over the Serverless documentations, I came across this page which would be the right place to surface details- https://docs.elastic.co/serverless/security/security-billing

### Related links / assets

Please include each of the following, if applicable:
Doc URL: https://docs.elastic.co/serverless/security/security-billing
Subject matter expert: @CohenIdo @smriti0321 
Figma link(s):
Github epic link(s): https://github.com/elastic/security-team/issues/8970
Github issue link(s): https://github.com/elastic/security-team/issues/9062

### Which documentation set needs improvement?

Serverless only

### Software version

Serverless latest version

### Collaborators

PM: @smriti0321 
Designer: 
Developer: @CohenIdo @maximkh
Others (if applicable): @kfirpeled @tehilashn 

### Timeline / deliverables

25th May 2024

### PR DESCRIPTION
**Pull request by benironside**
Resolves #5166. Updates the serverless billing page to reflect the changes to which cloud assets we will bill for, for customers using serverless.

Preview: [Security billing dimensions](https://docs-elastic-n8y8mrf1e-elastic-dev.vercel.app/serverless/security/security-billing)

### CODE REVIEW

**Comment by benironside** [comment]
run docs-build

**Comment by elasticdocs** [comment]
:rocket: Built _co-preview-docs_ successfully!

* https://docs-elastic-kw4j6vtgk-elastic-dev.vercel.app

* Built with commit 93b25e80e236bbe3344f4b000725de99238ee28a

> Issues? Visit #next-docs in Slack

**Comment by smriti0321** [comment]
@benironside Please review the additional ask in the comment- https://github.com/elastic/security-docs/issues/5166#issuecomment-2141699465 and help with latest information in document. Thank you!

File: docs/serverless/billing.mdx [hunk 1]
```diff
@@ -32,6 +32,29 @@ Cloud Protection is an _optional_ add-on to Security Analytics that provides val
...
+    - **Azure:** SQL database, Cosmos DB, Synapse Analytics
+    - **GCP:** Cloud SQL, Firestore, BigQuery
+
+For <DocLink id="serverlessSecurityKspm" text="KSPM"/>, billing is per monitored Kubernetes nodes (per `agent.id`).
+
+For <DocLink id="serverlessSecurityVulnManagementOverview" text="CNVM"/>, billing is per monitored cloud asset (per `cloud.instance.id`).
+
+For <DocLink id="serverlessSecurityD4cOverview" text="D4C"/>, billing is per protected agent (per `agent.id`).
+
+Note that logs, events, alerts, and configuration data ingested into your security project are billed using the **Ingest** and **Retention** pricing described above.
```
Comment (nastasha-solomon) [inline docs/serverless/billing.mdx:58]: Minor suggestion if you want to keep the same structure going.

```suggestion
Logs, events, alerts, and configuration data ingested into your security project are billed using the **Ingest** and **Retention** pricing described above.
```

**Review by nastasha-solomon — APPROVED** [review]
Looks great! Left some suggestions for your consideration. : )

File: docs/serverless/billing.mdx [hunk 2]
```diff
@@ -32,6 +32,29 @@ Cloud Protection is an _optional_ add-on to Security Analytics that provides val
 * **Cloud Protection Essentials** — Protects your cloud workloads, continuously tracks posture of your cloud assets, and helps you manage risks by detecting configuration issues per CIS benchmarks.
 * **Cloud Protection Complete** — Adds response capabilities and configuration drift prevention for Cloud Workloads.
 
-You pay based on the number of protected cloud workload and other cloud assets you configure for use with Elastic Cloud Security. Note that logs, events, alerts, and configuration data ingested into your security project are billed using the **Ingest** and **Retention** pricing described above.
+Your total cost depends on the number of protected cloud workloads and other billable cloud assets you configure for use with Elastic Cloud Security. 
 
-For detailed ((elastic-sec)) serverless project rates, check [Elastic Cloud pricing table](https://cloud.elastic.co/cloud-pricing-table?productType=serverless&project=security).
+For <DocLink id="serverlessSecurityCspm" text="CSPM"/> billing is based on how many billable resources (`resource.id`s) you monitor. The following types of assets are considered billable:
```
Comment (nastasha-solomon) [inline docs/serverless/billing.mdx:37]:
```suggestion
For <DocLink id="serverlessSecurityCspm" text="CSPM"/>, billing is based on how many billable resources (`resource.id`s) you monitor. The following types of assets are considered billable:
```

**Review by nastasha-solomon — APPROVED** [review]

**Pull request merged**